docs/rules/no-angle-bracket-type-assertion.md

```diff
@@ -0,0 +1,32 @@
+# Enforces the use of `as Type` assertions instead of `<Type>` assertions.
+
+Typescript disallows the use of `<Type>` assertions in `.tsx` because of the similitude with  
```

Comment:
Please use "PascalCase" for "TypeScript".

"Similitude" is an epic, but somewhat archaic word, I feel it would be much more common to use "similarity" 😄